config/routes.rb

```diff
@@ -2765,6 +2765,62 @@
                save_post
     },
 
+    :resource_pool_cloud      => {
```

Comment:
did the resource_pools route go away with this change?  I was expecting a delete.